biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -237,9 +259,60 @@ subroutine disturbance_rates( site_in, bc_in)
 
           currentCohort => currentCohort%taller
        end do
+
+       ! ! Counter of secondary patch used in logging debt calculation
+       ! if ( currentPatch%anthro_disturbance_label .eq. secondaryforest ) then
+       !    patch_no_secondary = patch_no_secondary + 1
+       ! end if
+
        currentPatch => currentPatch%younger
     end do
 
```

Comment:
it would be nice to encapsulate the following code in a routine. Usually its not a big deal, but disturbance code is some of our more complicated code, so its nice to keep that code as slim as possible.

Reply:
Will wrap up this piece of code.